frontend/src/metabase/components/AdminPaneLayout.styled.jsx

```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
+
+export const Container = styled.section`
+  padding-left: 1rem;
+  padding-right: 1rem;
+`;
+
+export const HeadingContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  gap: 0 1rem;
```

Comment:
[`gap` is still not fully supported](https://caniuse.com/?search=gap) (88%), but I think it's a good compromise than trying to mess around with margin and padding.

## This is what it will look like when browsers don't support `gap`
![image](https://user-images.githubusercontent.com/1937582/160374512-7ce4d123-59ee-407f-8a08-c74079438ec5.png)

## This is when it's supported
![image](https://user-images.githubusercontent.com/1937582/160374565-bfd81d52-c67a-4903-947b-1d5d8fe12f42.png)